test/sqlsrv/sqlsrv_streams_null_nchar.phpt

```diff
@@ -0,0 +1,111 @@
+--TEST--
+Populate different test tables with unicode character fields using null stream data as inputs
+--FILE--
+﻿﻿<?php
+include 'tools.inc';
+
+function NullStream_Char2Stream($conn)
+{
+    $tableName = GetTempTableName();
+    
+    // create another test table a char(512) column
+    $stmt = sqlsrv_query($conn, "CREATE TABLE $tableName ([c1_int] int, [c2_nchar] nchar(512))");
+    sqlsrv_free_stmt($stmt);
+
+    $fname = null;
+    $stmt = sqlsrv_query($conn, "INSERT INTO $tableName (c1_int, c2_nchar) VALUES (?, ?)", array(-187518515, &$fname));
```

Comment:
Can't we just create one table with all these types, instead of so much duplication?